app.rb

```diff
@@ -1,9 +1,30 @@
 require 'sinatra/base'
+require 'sinatra/reloader'
+require_relative './lib/peep'
+
 
 class Chitter < Sinatra::Base
+
+  configure :development do
+    register Sinatra::Reloader
+  end
+
   get '/test' do
     'Test page'
   end
 
+  get '/' do
+    erb:'index'
+  end
+
+  get '/peep' do
+  peeps = 
+    [
+      "this is my first peep!",
+      "this is my second peep!",
+      "this is my third peep!"
+    ]
```

Comment:
At the end of `get "/peep" do` you probably want to render a template like `erb :index`.  At the moment your test probably will pass but you'll just see the array on the screen, rather than a nice list of peeps